changelog/unreleased/issue-3183

```diff
@@ -0,0 +1,6 @@
+Bugfix: Allow specify --tag option several times and the resulting backup must contain all the specified tags.
```

Comment:
Please remove this changelog file and add a link to this PR to the changelog `issue-2688`. The bug hasn't been part of a released restic version. There's no need to inform regular users about bugs that were introduced and fixed between releases.